package.json

```diff
@@ -32,7 +32,7 @@
     }
   },
   "engines": {
-    "node": ">= 16"
```

Comment:
just curious, what changes in here broke node 16 compatibility? 

ideally I'd like to support 16 until EOL, but it's not a deal breaker for me.

Reply:
16 is EOL in 2 months and 4 weeks, will move it down to 16